addons/web_editor/static/src/xml/snippets.xml

```diff
@@ -50,9 +50,12 @@
         </we-customizeblock-options>
     </t>
     <t t-name="web_editor.outdated_block_message">
-        <we-alert>
-            <we-title>This block is outdated</we-title>
-            <span>To make changes, drop this block and use the new options in the last version.</span>
+        <we-alert class="o_outdated_block position-fixed d-flex flex-column h-100 p-3 pt-5 align-items-center text-center text-white">
+            <we-title>The <t t-out="name"/> block is outdated.</we-title>
+            <span>You might not be able to customize it anymore.</span>
+            <we-button class="o_snippet_replace o_we_bg_brand_primary py-2 my-4 border-0">REPLACE BY NEW VERSION</we-button>
+            <span>You can still access the block options but it might be ineffective.</span>
```

Comment:
`.pot` updates :wink:

Reply:
Isn't it only needed in stable ? I think we don't have to do that when we are in master :)